src/main.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file main.cpp
+ * @author Oleksandr Ananiev (alexander.ananiev@sigma.sofware)
+ * @brief Source file for main body of program
+ * @version 0.1
+ * @date 2021-02-04
+ * 
+ * @copyright Copyright (c) 2021
+ * 
+ */
+
+/**
+ * @defgroup Core
+ */
+
+#include <iostream>
+#include "TCP_Server.hpp"
+
+const uint16_t PORT = 3390;   ///< Port number on which the server will be created

```

Comment:
Let's make it a command-line argument. You can make 3390 as a default port number if no was set by the user.
So you will open your application with a port number as a command-line argument